export/planloader/planloader.h

```diff
@@ -0,0 +1,38 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include <substrait/common/Io.h>
+
+extern "C" {
+
+using SerializedPlan = struct {
+  // If set, contains a serialized ::substrait::proto::Plan object.
+  char *buffer;
+  // If buffer is set, this is the size of the buffer.
+  int32_t size;
+  // If null the buffer is valid, otherwise this points to an error message.
+  char *errorMessage;
+};
+
+// Since this is actually C code, stick to C style names for exporting.
+// NOLINTBEGIN(readability-identifier-naming)
```

Comment:
Hmmm, we have C style method names but Java style field names and argument names.  Should we make those snake_case as well?

Reply:
Done.